libs/platform/src/lib/value-help-dialog/components/define-tab/define-tab.component.ts

```diff
@@ -38,6 +40,11 @@ class ExtendedExcludedEntity extends VhdExcludedEntity {
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DefineTabComponent<T> extends VhdBaseTab implements OnChanges {
+  protected defaultId = `fd-title-id-${randomId++}`;
+
+  @Input()
+  titleId: string = this.defaultId;
```

Comment:
Seems like it can be shortened to 

```
@Input()
titleId: string = `fd-title-id-${randomId++}`;
```

Please use `unique` instead of `random` so it will be `titleUniqueId`.

Reply:
this will fail in lint so its has to be @Input() titleId: string = this.defaultId; like this.

Reply:
It shouldn't fail. At least we have a lot of similar places, for example look at the `platform/src/lib/table/table.component.ts`.